<?xml version="1.0" encoding="utf-8" standalone="no"?>
<test-run id="2" testcasecount="52" result="Failed" total="17" passed="5" failed="12" inconclusive="0" skipped="0" asserts="29" engine-version="3.10.0.0" clr-version="4.0.30319.42000" start-time="2019-06-11 12:26:17Z" end-time="2019-06-11 12:28:24Z" duration="126.685812">
  <command-line><![CDATA[C:\Users\itancau\.nuget\packages\nunit.consolerunner\3.10.0\tools\nunit3-console.exe  C:\Training\CSharpAutomationTraining\TestProject\bin\Debug\TestProject.dll --test=TestProject.Course2 -result=TestResults.xml;format=nunit3]]></command-line>
  <filter>
    <test>TestProject.Course2</test>
  </filter>
  <test-suite type="Assembly" id="0-1059" name="TestProject.dll" fullname="TestProject.dll" runstate="Runnable" testcasecount="52" result="Failed" site="Child" start-time="2019-06-11 12:26:18Z" end-time="2019-06-11 12:28:24Z" duration="125.045563" total="17" passed="5" failed="12" warnings="0" inconclusive="0" skipped="0" asserts="29">
    <environment framework-version="3.11.0.0" clr-version="4.0.30319.42000" os-version="Microsoft Windows NT 10.0.17763.0" platform="Win32NT" cwd="C:\Training\CSharpAutomationTraining\TestProject\NunitConsoleRunner" machine-name="EN610820" user="itancau" user-domain="ENDAVA" culture="en-US" uiculture="en-US" os-architecture="x64" />
    <settings>
      <setting name="DisposeRunners" value="True" />
      <setting name="WorkDirectory" value="C:\Training\CSharpAutomationTraining\TestProject\NunitConsoleRunner" />
      <setting name="ImageRuntimeVersion" value="4.0.30319" />
      <setting name="ImageTargetFrameworkName" value=".NETFramework,Version=v4.6.1" />
      <setting name="ImageRequiresX86" value="False" />
      <setting name="ImageRequiresDefaultAppDomainAssemblyResolver" value="False" />
      <setting name="NumberOfTestWorkers" value="4" />
    </settings>
    <properties>
      <property name="_PID" value="24572" />
      <property name="_APPDOMAIN" value="domain-" />
    </properties>
    <failure>
      <message><![CDATA[One or more child tests had errors]]></message>
    </failure>
    <test-suite type="TestSuite" id="0-1060" name="TestProject" fullname="TestProject" runstate="Runnable" testcasecount="52" result="Failed" site="Child" start-time="2019-06-11 12:26:19Z" end-time="2019-06-11 12:28:24Z" duration="124.972595" total="17" passed="5" failed="12" warnings="0" inconclusive="0" skipped="0" asserts="29">
      <failure>
        <message><![CDATA[One or more child tests had errors]]></message>
      </failure>
      <test-suite type="TestSuite" id="0-1063" name="Course2" fullname="TestProject.Course2" runstate="Runnable" testcasecount="17" result="Failed" site="Child" start-time="2019-06-11 12:26:19Z" end-time="2019-06-11 12:28:24Z" duration="124.971459" total="17" passed="5" failed="12" warnings="0" inconclusive="0" skipped="0" asserts="29">
        <failure>
          <message><![CDATA[One or more child tests had errors]]></message>
        </failure>
        <test-suite type="TestSuite" id="0-1065" name="DashboardPageTests" fullname="TestProject.Course2.DashboardPageTests" runstate="Runnable" testcasecount="6" result="Failed" site="Child" start-time="2019-06-11 12:26:19Z" end-time="2019-06-11 12:27:11Z" duration="52.007273" total="6" passed="5" failed="1" warnings="0" inconclusive="0" skipped="0" asserts="10">
          <failure>
            <message><![CDATA[One or more child tests had errors]]></message>
          </failure>
          <test-suite type="TestFixture" id="0-1033" name="DashboardPageTests" fullname="TestProject.Course2.DashboardPageTests.DashboardPageTests" classname="TestProject.Course2.DashboardPageTests.DashboardPageTests" runstate="Runnable" testcasecount="6" result="Failed" site="Child" start-time="2019-06-11 12:26:19Z" end-time="2019-06-11 12:27:11Z" duration="51.991157" total="6" passed="5" failed="1" warnings="0" inconclusive="0" skipped="0" asserts="10">
            <failure>
              <message><![CDATA[One or more child tests had errors]]></message>
            </failure>
            <test-case id="0-1037" name="EditPersonalInformationSuccesful" fullname="TestProject.Course2.DashboardPageTests.DashboardPageTests.EditPersonalInformationSuccesful" methodname="EditPersonalInformationSuccesful" classname="TestProject.Course2.DashboardPageTests.DashboardPageTests" runstate="Runnable" seed="1928223533" result="Passed" start-time="2019-06-11 12:26:19Z" end-time="2019-06-11 12:26:34Z" duration="15.073247" asserts="1" />
            <test-case id="0-1039" name="FooterLinksAreDisplayed" fullname="TestProject.Course2.DashboardPageTests.DashboardPageTests.FooterLinksAreDisplayed" methodname="FooterLinksAreDisplayed" classname="TestProject.Course2.DashboardPageTests.DashboardPageTests" runstate="Runnable" seed="1546040109" result="Passed" start-time="2019-06-11 12:26:34Z" end-time="2019-06-11 12:26:40Z" duration="6.214798" asserts="3" />
            <test-case id="0-1034" name="HeaderItemsDisplayed" fullname="TestProject.Course2.DashboardPageTests.DashboardPageTests.HeaderItemsDisplayed" methodname="HeaderItemsDisplayed" classname="TestProject.Course2.DashboardPageTests.DashboardPageTests" runstate="Runnable" seed="648735428" result="Passed" start-time="2019-06-11 12:26:40Z" end-time="2019-06-11 12:26:47Z" duration="6.289590" asserts="3" />
            <test-case id="0-1036" name="PageHeadlingTitleOk" fullname="TestProject.Course2.DashboardPageTests.DashboardPageTests.PageHeadlingTitleOk" methodname="PageHeadlingTitleOk" classname="TestProject.Course2.DashboardPageTests.DashboardPageTests" runstate="Runnable" seed="598708104" result="Passed" start-time="2019-06-11 12:26:47Z" end-time="2019-06-11 12:26:53Z" duration="6.459569" asserts="1" />
            <test-case id="0-1035" name="PageTitleOk" fullname="TestProject.Course2.DashboardPageTests.DashboardPageTests.PageTitleOk" methodname="PageTitleOk" classname="TestProject.Course2.DashboardPageTests.DashboardPageTests" runstate="Runnable" seed="712151827" result="Passed" start-time="2019-06-11 12:26:53Z" end-time="2019-06-11 12:26:59Z" duration="6.319118" asserts="1" />
            <test-case id="0-1038" name="SuccessfulLogOut" fullname="TestProject.Course2.DashboardPageTests.DashboardPageTests.SuccessfulLogOut" methodname="SuccessfulLogOut" classname="TestProject.Course2.DashboardPageTests.DashboardPageTests" runstate="Runnable" seed="240977595" result="Failed" start-time="2019-06-11 12:26:59Z" end-time="2019-06-11 12:27:08Z" duration="8.728672" asserts="1">
              <failure>
                <message><![CDATA[  Wrong page title
  Expected string length 9 but was 14. Strings differ at index 0.
  Expected: "Home Page", ignoring case
  But was:  "Dashboard page"
  -----------^
]]></message>
                <stack-trace><![CDATA[   at TestProject.Course2.POM.HomePagePOM.CheckPageTitle() in C:\Training\CSharpAutomationTraining\TestProject\Course2\POM\HomePagePOM.cs:line 45
   at TestProject.Course2.DashboardPageTests.DashboardPageTests.SuccessfulLogOut() in C:\Training\CSharpAutomationTraining\TestProject\Course2\Tests\DashboardPageTests.cs:line 41
]]></stack-trace>
              </failure>
              <assertions>
                <assertion result="Failed">
                  <message><![CDATA[  Wrong page title
  Expected string length 9 but was 14. Strings differ at index 0.
  Expected: "Home Page", ignoring case
  But was:  "Dashboard page"
  -----------^
]]></message>
                  <stack-trace><![CDATA[   at TestProject.Course2.POM.HomePagePOM.CheckPageTitle() in C:\Training\CSharpAutomationTraining\TestProject\Course2\POM\HomePagePOM.cs:line 45
   at TestProject.Course2.DashboardPageTests.DashboardPageTests.SuccessfulLogOut() in C:\Training\CSharpAutomationTraining\TestProject\Course2\Tests\DashboardPageTests.cs:line 41
]]></stack-trace>
                </assertion>
              </assertions>
            </test-case>
          </test-suite>
        </test-suite>
        <test-suite type="TestSuite" id="0-1064" name="HomePageTests" fullname="TestProject.Course2.HomePageTests" runstate="Runnable" testcasecount="11" result="Failed" site="Child" start-time="2019-06-11 12:26:19Z" end-time="2019-06-11 12:28:24Z" duration="124.971388" total="11" passed="0" failed="11" warnings="0" inconclusive="0" skipped="0" asserts="19">
          <failure>
            <message><![CDATA[One or more child tests had errors]]></message>
          </failure>
          <test-suite type="TestFixture" id="0-1021" name="HomePageTests" fullname="TestProject.Course2.HomePageTests.HomePageTests" classname="TestProject.Course2.HomePageTests.HomePageTests" runstate="Runnable" testcasecount="11" result="Failed" label="Error" site="TearDown" start-time="2019-06-11 12:27:11Z" end-time="2019-06-11 12:28:24Z" duration="72.947392" total="11" passed="0" failed="11" warnings="0" inconclusive="0" skipped="0" asserts="19">
            <failure>
              <message><![CDATA[One or more child tests had errors
TearDown : System.IO.IOException : The process cannot access the file 'C:\Training\CSharpAutomationTraining\TestProject\Course2\Reports\Results\ReportFiles\11.06.2019-03.26.19.html' because it is being used by another process.]]></message>
              <stack-trace><![CDATA[--TearDown
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   at System.IO.File.InternalWriteAllText(String path, String contents, Encoding encoding, Boolean checkHost)
   at AventStack.ExtentReports.Reporter.ExtentHtmlReporter.Flush()
   at AventStack.ExtentReports.Model.Report.<>c.<NotifyReporters>b__29_1(IExtentReporter x)
   at System.Collections.Generic.List`1.ForEach(Action`1 action)
   at AventStack.ExtentReports.Model.Report.NotifyReporters()
   at AventStack.ExtentReports.Model.Report.Flush()
   at AventStack.ExtentReports.ExtentReports.Flush()
   at TestProject.Course2.Reports.Reporter.EndReporting() in C:\Training\CSharpAutomationTraining\TestProject\Course2\Reports\Reporter.cs:line 41
   at TestProject.Course2.Base.BasePage.AfterTestClass() in C:\Training\CSharpAutomationTraining\TestProject\Course2\Base\BasePage.cs:line 33]]></stack-trace>
            </failure>
            <test-case id="0-1025" name="DefaultLogInDetailsDisplayed" fullname="TestProject.Course2.HomePageTests.HomePageTests.DefaultLogInDetailsDisplayed" methodname="DefaultLogInDetailsDisplayed" classname="TestProject.Course2.HomePageTests.HomePageTests" runstate="Runnable" seed="1025822675" result="Failed" label="Error" start-time="2019-06-11 12:27:11Z" end-time="2019-06-11 12:27:21Z" duration="10.477699" asserts="2">
              <failure>
                <message><![CDATA[TearDown : System.IO.IOException : The process cannot access the file 'C:\Training\CSharpAutomationTraining\TestProject\Course2\Reports\Results\ReportFiles\11.06.2019-03.26.19.html' because it is being used by another process.]]></message>
                <stack-trace><![CDATA[--TearDown
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   at System.IO.File.InternalWriteAllText(String path, String contents, Encoding encoding, Boolean checkHost)
   at AventStack.ExtentReports.Reporter.ExtentHtmlReporter.Flush()
   at AventStack.ExtentReports.Model.Report.<>c.<NotifyReporters>b__29_1(IExtentReporter x)
   at System.Collections.Generic.List`1.ForEach(Action`1 action)
   at AventStack.ExtentReports.Model.Report.NotifyReporters()
   at AventStack.ExtentReports.Model.Report.Flush()
   at AventStack.ExtentReports.ExtentReports.Flush()
   at TestProject.Course2.Reports.Reporter.EndTest() in C:\Training\CSharpAutomationTraining\TestProject\Course2\Reports\Reporter.cs:line 73
   at TestProject.Course2.Base.BasePage.AfterEachTest() in C:\Training\CSharpAutomationTraining\TestProject\Course2\Base\BasePage.cs:line 40]]></stack-trace>
              </failure>
            </test-case>
            <test-case id="0-1032" name="FooterLinksAreDisplayed" fullname="TestProject.Course2.HomePageTests.HomePageTests.FooterLinksAreDisplayed" methodname="FooterLinksAreDisplayed" classname="TestProject.Course2.HomePageTests.HomePageTests" runstate="Runnable" seed="1010985976" result="Failed" label="Error" start-time="2019-06-11 12:27:21Z" end-time="2019-06-11 12:27:27Z" duration="5.946023" asserts="3">
              <failure>
                <message><![CDATA[TearDown : System.IO.IOException : The process cannot access the file 'C:\Training\CSharpAutomationTraining\TestProject\Course2\Reports\Results\ReportFiles\11.06.2019-03.26.19.html' because it is being used by another process.]]></message>
                <stack-trace><![CDATA[--TearDown
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   at System.IO.File.InternalWriteAllText(String path, String contents, Encoding encoding, Boolean checkHost)
   at AventStack.ExtentReports.Reporter.ExtentHtmlReporter.Flush()
   at AventStack.ExtentReports.Model.Report.<>c.<NotifyReporters>b__29_1(IExtentReporter x)
   at System.Collections.Generic.List`1.ForEach(Action`1 action)
   at AventStack.ExtentReports.Model.Report.NotifyReporters()
   at AventStack.ExtentReports.Model.Report.Flush()
   at AventStack.ExtentReports.ExtentReports.Flush()
   at TestProject.Course2.Reports.Reporter.EndTest() in C:\Training\CSharpAutomationTraining\TestProject\Course2\Reports\Reporter.cs:line 73
   at TestProject.Course2.Base.BasePage.AfterEachTest() in C:\Training\CSharpAutomationTraining\TestProject\Course2\Base\BasePage.cs:line 40]]></stack-trace>
              </failure>
            </test-case>
            <test-case id="0-1022" name="HeaderItemsDisplayed" fullname="TestProject.Course2.HomePageTests.HomePageTests.HeaderItemsDisplayed" methodname="HeaderItemsDisplayed" classname="TestProject.Course2.HomePageTests.HomePageTests" runstate="Runnable" seed="1129566981" result="Failed" label="Error" start-time="2019-06-11 12:27:27Z" end-time="2019-06-11 12:27:35Z" duration="7.843469" asserts="3">
              <failure>
                <message><![CDATA[TearDown : System.IO.IOException : The process cannot access the file 'C:\Training\CSharpAutomationTraining\TestProject\Course2\Reports\Results\ReportFiles\11.06.2019-03.26.19.html' because it is being used by another process.]]></message>
                <stack-trace><![CDATA[--TearDown
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   at System.IO.File.InternalWriteAllText(String path, String contents, Encoding encoding, Boolean checkHost)
   at AventStack.ExtentReports.Reporter.ExtentHtmlReporter.Flush()
   at AventStack.ExtentReports.Model.Report.<>c.<NotifyReporters>b__29_1(IExtentReporter x)
   at System.Collections.Generic.List`1.ForEach(Action`1 action)
   at AventStack.ExtentReports.Model.Report.NotifyReporters()
   at AventStack.ExtentReports.Model.Report.Flush()
   at AventStack.ExtentReports.ExtentReports.Flush()
   at TestProject.Course2.Reports.Reporter.EndTest() in C:\Training\CSharpAutomationTraining\TestProject\Course2\Reports\Reporter.cs:line 73
   at TestProject.Course2.Base.BasePage.AfterEachTest() in C:\Training\CSharpAutomationTraining\TestProject\Course2\Base\BasePage.cs:line 40]]></stack-trace>
              </failure>
            </test-case>
            <test-case id="0-1028" name="InvalidEmailFormatErrorMessage" fullname="TestProject.Course2.HomePageTests.HomePageTests.InvalidEmailFormatErrorMessage" methodname="InvalidEmailFormatErrorMessage" classname="TestProject.Course2.HomePageTests.HomePageTests" runstate="Runnable" seed="1299402720" result="Failed" label="Error" start-time="2019-06-11 12:27:35Z" end-time="2019-06-11 12:27:41Z" duration="6.308319" asserts="1">
              <failure>
                <message><![CDATA[TearDown : System.IO.IOException : The process cannot access the file 'C:\Training\CSharpAutomationTraining\TestProject\Course2\Reports\Results\ReportFiles\11.06.2019-03.26.19.html' because it is being used by another process.]]></message>
                <stack-trace><![CDATA[--TearDown
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   at System.IO.File.InternalWriteAllText(String path, String contents, Encoding encoding, Boolean checkHost)
   at AventStack.ExtentReports.Reporter.ExtentHtmlReporter.Flush()
   at AventStack.ExtentReports.Model.Report.<>c.<NotifyReporters>b__29_1(IExtentReporter x)
   at System.Collections.Generic.List`1.ForEach(Action`1 action)
   at AventStack.ExtentReports.Model.Report.NotifyReporters()
   at AventStack.ExtentReports.Model.Report.Flush()
   at AventStack.ExtentReports.ExtentReports.Flush()
   at TestProject.Course2.Reports.Reporter.EndTest() in C:\Training\CSharpAutomationTraining\TestProject\Course2\Reports\Reporter.cs:line 73
   at TestProject.Course2.Base.BasePage.AfterEachTest() in C:\Training\CSharpAutomationTraining\TestProject\Course2\Base\BasePage.cs:line 40]]></stack-trace>
              </failure>
            </test-case>
            <test-case id="0-1029" name="InvalidEmailOrPasswordErrorMessage" fullname="TestProject.Course2.HomePageTests.HomePageTests.InvalidEmailOrPasswordErrorMessage" methodname="InvalidEmailOrPasswordErrorMessage" classname="TestProject.Course2.HomePageTests.HomePageTests" runstate="Runnable" seed="1639494039" result="Failed" label="Error" start-time="2019-06-11 12:27:41Z" end-time="2019-06-11 12:27:47Z" duration="6.242518" asserts="1">
              <failure>
                <message><![CDATA[TearDown : System.IO.IOException : The process cannot access the file 'C:\Training\CSharpAutomationTraining\TestProject\Course2\Reports\Results\ReportFiles\11.06.2019-03.26.19.html' because it is being used by another process.]]></message>
                <stack-trace><![CDATA[--TearDown
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   at System.IO.File.InternalWriteAllText(String path, String contents, Encoding encoding, Boolean checkHost)
   at AventStack.ExtentReports.Reporter.ExtentHtmlReporter.Flush()
   at AventStack.ExtentReports.Model.Report.<>c.<NotifyReporters>b__29_1(IExtentReporter x)
   at System.Collections.Generic.List`1.ForEach(Action`1 action)
   at AventStack.ExtentReports.Model.Report.NotifyReporters()
   at AventStack.ExtentReports.Model.Report.Flush()
   at AventStack.ExtentReports.ExtentReports.Flush()
   at TestProject.Course2.Reports.Reporter.EndTest() in C:\Training\CSharpAutomationTraining\TestProject\Course2\Reports\Reporter.cs:line 73
   at TestProject.Course2.Base.BasePage.AfterEachTest() in C:\Training\CSharpAutomationTraining\TestProject\Course2\Base\BasePage.cs:line 40]]></stack-trace>
              </failure>
            </test-case>
            <test-case id="0-1026" name="LogInFieldsDisplayed" fullname="TestProject.Course2.HomePageTests.HomePageTests.LogInFieldsDisplayed" methodname="LogInFieldsDisplayed" classname="TestProject.Course2.HomePageTests.HomePageTests" runstate="Runnable" seed="771549603" result="Failed" label="Error" start-time="2019-06-11 12:27:47Z" end-time="2019-06-11 12:27:53Z" duration="5.738924" asserts="3">
              <failure>
                <message><![CDATA[TearDown : System.IO.IOException : The process cannot access the file 'C:\Training\CSharpAutomationTraining\TestProject\Course2\Reports\Results\ReportFiles\11.06.2019-03.26.19.html' because it is being used by another process.]]></message>
                <stack-trace><![CDATA[--TearDown
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   at System.IO.File.InternalWriteAllText(String path, String contents, Encoding encoding, Boolean checkHost)
   at AventStack.ExtentReports.Reporter.ExtentHtmlReporter.Flush()
   at AventStack.ExtentReports.Model.Report.<>c.<NotifyReporters>b__29_1(IExtentReporter x)
   at System.Collections.Generic.List`1.ForEach(Action`1 action)
   at AventStack.ExtentReports.Model.Report.NotifyReporters()
   at AventStack.ExtentReports.Model.Report.Flush()
   at AventStack.ExtentReports.ExtentReports.Flush()
   at TestProject.Course2.Reports.Reporter.EndTest() in C:\Training\CSharpAutomationTraining\TestProject\Course2\Reports\Reporter.cs:line 73
   at TestProject.Course2.Base.BasePage.AfterEachTest() in C:\Training\CSharpAutomationTraining\TestProject\Course2\Base\BasePage.cs:line 40]]></stack-trace>
              </failure>
            </test-case>
            <test-case id="0-1027" name="MissingLogInDetailsErrorMessage" fullname="TestProject.Course2.HomePageTests.HomePageTests.MissingLogInDetailsErrorMessage" methodname="MissingLogInDetailsErrorMessage" classname="TestProject.Course2.HomePageTests.HomePageTests" runstate="Runnable" seed="392931365" result="Failed" label="Error" start-time="2019-06-11 12:27:53Z" end-time="2019-06-11 12:27:59Z" duration="6.147762" asserts="2">
              <failure>
                <message><![CDATA[TearDown : System.IO.IOException : The process cannot access the file 'C:\Training\CSharpAutomationTraining\TestProject\Course2\Reports\Results\ReportFiles\11.06.2019-03.26.19.html' because it is being used by another process.]]></message>
                <stack-trace><![CDATA[--TearDown
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   at System.IO.File.InternalWriteAllText(String path, String contents, Encoding encoding, Boolean checkHost)
   at AventStack.ExtentReports.Reporter.ExtentHtmlReporter.Flush()
   at AventStack.ExtentReports.Model.Report.<>c.<NotifyReporters>b__29_1(IExtentReporter x)
   at System.Collections.Generic.List`1.ForEach(Action`1 action)
   at AventStack.ExtentReports.Model.Report.NotifyReporters()
   at AventStack.ExtentReports.Model.Report.Flush()
   at AventStack.ExtentReports.ExtentReports.Flush()
   at TestProject.Course2.Reports.Reporter.EndTest() in C:\Training\CSharpAutomationTraining\TestProject\Course2\Reports\Reporter.cs:line 73
   at TestProject.Course2.Base.BasePage.AfterEachTest() in C:\Training\CSharpAutomationTraining\TestProject\Course2\Base\BasePage.cs:line 40]]></stack-trace>
              </failure>
            </test-case>
            <test-case id="0-1024" name="PageHeadlingTitleOk" fullname="TestProject.Course2.HomePageTests.HomePageTests.PageHeadlingTitleOk" methodname="PageHeadlingTitleOk" classname="TestProject.Course2.HomePageTests.HomePageTests" runstate="Runnable" seed="1099179358" result="Failed" label="Error" start-time="2019-06-11 12:27:59Z" end-time="2019-06-11 12:28:05Z" duration="5.841617" asserts="1">
              <failure>
                <message><![CDATA[TearDown : System.IO.IOException : The process cannot access the file 'C:\Training\CSharpAutomationTraining\TestProject\Course2\Reports\Results\ReportFiles\11.06.2019-03.26.19.html' because it is being used by another process.]]></message>
                <stack-trace><![CDATA[--TearDown
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   at System.IO.File.InternalWriteAllText(String path, String contents, Encoding encoding, Boolean checkHost)
   at AventStack.ExtentReports.Reporter.ExtentHtmlReporter.Flush()
   at AventStack.ExtentReports.Model.Report.<>c.<NotifyReporters>b__29_1(IExtentReporter x)
   at System.Collections.Generic.List`1.ForEach(Action`1 action)
   at AventStack.ExtentReports.Model.Report.NotifyReporters()
   at AventStack.ExtentReports.Model.Report.Flush()
   at AventStack.ExtentReports.ExtentReports.Flush()
   at TestProject.Course2.Reports.Reporter.EndTest() in C:\Training\CSharpAutomationTraining\TestProject\Course2\Reports\Reporter.cs:line 73
   at TestProject.Course2.Base.BasePage.AfterEachTest() in C:\Training\CSharpAutomationTraining\TestProject\Course2\Base\BasePage.cs:line 40]]></stack-trace>
              </failure>
            </test-case>
            <test-case id="0-1023" name="PageTitleOk" fullname="TestProject.Course2.HomePageTests.HomePageTests.PageTitleOk" methodname="PageTitleOk" classname="TestProject.Course2.HomePageTests.HomePageTests" runstate="Runnable" seed="521383395" result="Failed" label="Error" start-time="2019-06-11 12:28:05Z" end-time="2019-06-11 12:28:11Z" duration="5.827989" asserts="1">
              <failure>
                <message><![CDATA[TearDown : System.IO.IOException : The process cannot access the file 'C:\Training\CSharpAutomationTraining\TestProject\Course2\Reports\Results\ReportFiles\11.06.2019-03.26.19.html' because it is being used by another process.]]></message>
                <stack-trace><![CDATA[--TearDown
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   at System.IO.File.InternalWriteAllText(String path, String contents, Encoding encoding, Boolean checkHost)
   at AventStack.ExtentReports.Reporter.ExtentHtmlReporter.Flush()
   at AventStack.ExtentReports.Model.Report.<>c.<NotifyReporters>b__29_1(IExtentReporter x)
   at System.Collections.Generic.List`1.ForEach(Action`1 action)
   at AventStack.ExtentReports.Model.Report.NotifyReporters()
   at AventStack.ExtentReports.Model.Report.Flush()
   at AventStack.ExtentReports.ExtentReports.Flush()
   at TestProject.Course2.Reports.Reporter.EndTest() in C:\Training\CSharpAutomationTraining\TestProject\Course2\Reports\Reporter.cs:line 73
   at TestProject.Course2.Base.BasePage.AfterEachTest() in C:\Training\CSharpAutomationTraining\TestProject\Course2\Base\BasePage.cs:line 40]]></stack-trace>
              </failure>
            </test-case>
            <test-case id="0-1031" name="SuccesfulNavigateToWikiPage" fullname="TestProject.Course2.HomePageTests.HomePageTests.SuccesfulNavigateToWikiPage" methodname="SuccesfulNavigateToWikiPage" classname="TestProject.Course2.HomePageTests.HomePageTests" runstate="Runnable" seed="616139730" result="Failed" label="Error" start-time="2019-06-11 12:28:11Z" end-time="2019-06-11 12:28:17Z" duration="6.213735" asserts="1">
              <failure>
                <message><![CDATA[TearDown : System.IO.IOException : The process cannot access the file 'C:\Training\CSharpAutomationTraining\TestProject\Course2\Reports\Results\ReportFiles\11.06.2019-03.26.19.html' because it is being used by another process.]]></message>
                <stack-trace><![CDATA[--TearDown
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   at System.IO.File.InternalWriteAllText(String path, String contents, Encoding encoding, Boolean checkHost)
   at AventStack.ExtentReports.Reporter.ExtentHtmlReporter.Flush()
   at AventStack.ExtentReports.Model.Report.<>c.<NotifyReporters>b__29_1(IExtentReporter x)
   at System.Collections.Generic.List`1.ForEach(Action`1 action)
   at AventStack.ExtentReports.Model.Report.NotifyReporters()
   at AventStack.ExtentReports.Model.Report.Flush()
   at AventStack.ExtentReports.ExtentReports.Flush()
   at TestProject.Course2.Reports.Reporter.EndTest() in C:\Training\CSharpAutomationTraining\TestProject\Course2\Reports\Reporter.cs:line 73
   at TestProject.Course2.Base.BasePage.AfterEachTest() in C:\Training\CSharpAutomationTraining\TestProject\Course2\Base\BasePage.cs:line 40]]></stack-trace>
              </failure>
            </test-case>
            <test-case id="0-1030" name="SuccessfulLogIn" fullname="TestProject.Course2.HomePageTests.HomePageTests.SuccessfulLogIn" methodname="SuccessfulLogIn" classname="TestProject.Course2.HomePageTests.HomePageTests" runstate="Runnable" seed="755854809" result="Failed" label="Error" start-time="2019-06-11 12:28:17Z" end-time="2019-06-11 12:28:24Z" duration="6.328902" asserts="1">
              <failure>
                <message><![CDATA[TearDown : System.IO.IOException : The process cannot access the file 'C:\Training\CSharpAutomationTraining\TestProject\Course2\Reports\Results\ReportFiles\11.06.2019-03.26.19.html' because it is being used by another process.]]></message>
                <stack-trace><![CDATA[--TearDown
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   at System.IO.File.InternalWriteAllText(String path, String contents, Encoding encoding, Boolean checkHost)
   at AventStack.ExtentReports.Reporter.ExtentHtmlReporter.Flush()
   at AventStack.ExtentReports.Model.Report.<>c.<NotifyReporters>b__29_1(IExtentReporter x)
   at System.Collections.Generic.List`1.ForEach(Action`1 action)
   at AventStack.ExtentReports.Model.Report.NotifyReporters()
   at AventStack.ExtentReports.Model.Report.Flush()
   at AventStack.ExtentReports.ExtentReports.Flush()
   at TestProject.Course2.Reports.Reporter.EndTest() in C:\Training\CSharpAutomationTraining\TestProject\Course2\Reports\Reporter.cs:line 73
   at TestProject.Course2.Base.BasePage.AfterEachTest() in C:\Training\CSharpAutomationTraining\TestProject\Course2\Base\BasePage.cs:line 40]]></stack-trace>
              </failure>
            </test-case>
          </test-suite>
        </test-suite>
      </test-suite>
    </test-suite>
  </test-suite>
</test-run>